packages/cli/src/commands/setup/auth/providers/auth0.js

```diff
@@ -5,12 +5,27 @@ export const config = {
   domain: process.env.AUTH0_DOMAIN,
   client_id: process.env.AUTH0_CLIENT_ID,
   redirect_uri: process.env.AUTH0_REDIRECT_URI,
+
   // ** NOTE ** Storing tokens in browser local storage provides persistence across page refreshes and browser tabs.
   // However, if an attacker can achieve running JavaScript in the SPA using a cross-site scripting (XSS) attack,
   // they can retrieve the tokens stored in local storage.
   // https://auth0.com/docs/libraries/auth0-spa-js#change-storage-options
   cacheLocation: 'localstorage',
   audience: process.env.AUTH0_AUDIENCE,
+
```

Comment:
Could we make this a little less verbose DT? Large comment blocks like this can seem daunting right after. Perhaps just a the `@MARK` and the link to refresh token rotation doc

`useRefreshTokens: true` should also be commented out by default, as that's the default on Auth0 server side too

Reply:
I could put the info in the setup epilogue?